packages/node-bridge/launcher.js

```diff
@@ -3,6 +3,11 @@ const { createServer, Server } = require('http');
 const { isAbsolute } = require('path');
 const { Bridge } = require('./bridge.js');
 
+// TODO: is a 900-second timeout even useful for most cases?
+// Set to max timeout for ENT plans because we don't
+// know what plan the current user is on
+const SERVERLESS_FUNCTION_TIMEOUT = 900 * 1000; // 900 seconds
```

Comment:
We don't currently know the plan type when running `vc dev`. So, I set this timeout the max possible, which is 900 seconds for ENT.

Having a timeout at all is kinda nice, but I imagine this not being super useful until we get plan types in here.